src/styles/blocks/_details.scss

```
@@ -0,0 +1,191 @@
+.details {
+
+  padding: 10vw 6.25vw 12.5vw;
+
+  display: flex;
+  flex-direction: column;
+
+  &__title {
+    font-weight: 500;
+    text-align: center;
+    letter-spacing: -1.5px;
+
+    font-size: 32px;
+    line-height: 32px;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    margin-top: 48px;
+
+    text-align: left;
+
+    gap: 40px;
+
+    &-block {
+
+      &-images {
+        display: grid;
+        justify-content: space-between;
+
+        height: 40.62vw;
+        grid-template-columns: 40.62vw 40.62vw;
+
+        &-img {
+          background-size: cover;
+          background-repeat: no-repeat;
+          background-position: center;
+          height: 100%;
+
+          @extend %scale-on-hover;
+        }
+      }
+
+      &-title {
+        font-weight: 700;
+        margin-top: 16px;
+
+        font-size: 24px;
+        line-height: 140%;
+      }
+
+      &-description {
+        font-weight: 400;
+        margin-top: 8px;
+
+        font-size: 16px;
+        line-height: 140%;
+      }
+
+    }
+
+    @for $image from 1 through 6 {
+
+      .images-img--#{$image} {
+        background-image: url("../images/image#{$image + 3}.png");
```

```suggestion
        background-image: url("../images/image#{$image}.png");
```
Maybe you can just rename images instead of making this weird calculation?